### ISSUE TITLE
:decimal only for part of the world

### ISSUE THREAD

**Issue by jeanmartin** [bmichotte/ProMotion-XLForm#22]
Using the :decimal type in countries that use ',' as the decimal seperator (instead of '.') causes the value to be truncated (2,75 => 2.00).

According to http://stackoverflow.com/questions/11487850/decimal-point-in-uikeyboardtypedecimalpad-cant-be-used-in-mathematical-calculat a localizedScannerWithString (NSScanner) would need to be used. I however have no idea where to apply this.

**Comment by jeanmartin** [comment]
To reproduce simply switch the settings in your simulator to e.g. "Germany". Now the keypad for :decimal will display a ',' as the separator. Enter "2,74" and then leave the field, it will now show "2".

**Comment by bmichotte** [comment]
You're right. Looking at https://github.com/xmartlabs/XLForm/blob/master/XLForm/XL/Cell/XLFormTextFieldCell.m#L278 the only think it does is a `doubleValue`

I guess it's more a XLForm issue. Maybe you can report it on their repo.
